plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/transformers/api/IPlaceholderConfigurerSupport.java

```diff
@@ -0,0 +1,10 @@
+package org.hotswap.agent.plugin.spring.transformers.api;
+
+import org.springframework.util.StringValueResolver;
+
+import java.util.List;
+
+public interface IPlaceholderConfigurerSupport {
```

Comment:
Suggest to rename this interface name to 'StringValueResolverAware'.

Reply:
Maybe I will add more method in the future. So it is a common interface name.

Reply:
Pls. don't use 'I' to represent it's an interface,  it's regarding to code style.